### ISSUE TITLE
I finetune and test on the Ticket dataset and cannot get the same result reported in the paper.

### ISSUE THREAD

**Issue by chentianyangWHU** [clovaai/donut#154]
I finetune and test on the Ticket dataset and the result is ted_accuracy=0.9454, F1_accuracy=0.8686, which is lower than ted_accuracy=98.8/F1_accuracy=97.0 as reported in the paper. Could you please give some directions?
Here is the hyper-parameters I used:

resume_from_checkpoint_path: null 
result_path: "./result"
pretrained_model_name_or_path: "naver-clova-ix/donut-base"
dataset_name_or_paths: ["/data/zhazhenzhou/data/zhtrainticket"] # should be prepared from https://github.com/beacandler/EATEN
sort_json_key: True
train_batch_sizes: [8]   # [8]
val_batch_sizes: [1]
input_size: [960, 1280]
max_length: 256
align_long_axis: False
num_nodes: 1
seed: 2022
lr: 3e-5
warmup_steps: 300
num_training_samples_per_epoch: 1368    # 1368
max_epochs: 10
max_steps: -1
num_workers: 8
val_check_interval: 1.0
check_val_every_n_epoch: 1
gradient_clip_val: 1.0
verbose: True

Here is the traing process:
![image](https://user-images.githubusercontent.com/39194717/221463092-f421762a-5478-4d14-86de-30a9e70f5671.png)

**Comment by chentianyangWHU** [comment]
BTW, when I finetuned on CORD, the training loss will decrease to around 0.03, but on Ticket the training loss will not decrease after reaching 0.9. Is it normal？
